src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityRotorHolder.java

```diff
@@ -286,6 +292,8 @@ public void readFromNBT(NBTTagCompound data) {
         super.readFromNBT(data);
         this.inventory.deserializeNBT(data.getCompoundTag("inventory"));
         this.currentSpeed = data.getInteger("currentSpeed");
+        this.isRotorSpinning = data.getBoolean("Spinning");
+        this.frontFaceFree = data.getBoolean("FrontFree");
```

Comment:
These should check if those tags are not empty before attempting to read them, since we are adding the tags in this PR, and existing rotor holders will not have them

Reply:
if the tags are not there it will simply return false. And those values are updated in `update` anyways